lib/syslogger.rb

```diff
@@ -156,6 +157,14 @@ def clear_tags!
 
   protected
 
+  # stolen from ruby/logger -- formatter expects string representation of severity
+  # Severity label for logging (max 5 chars).
+  SEV_LABEL = %w(DEBUG INFO WARN ERROR FATAL ANY)
```

Comment:
Good point (or rather points). Given that there is no intrinsic order, and it is now different files, I converted to map.